frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.scenario.basic.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  flavorWorkloadConfigs,
+  vmwareSecondVMConfig,
+  vmwareVMConfig,
+  vmwareVMMultiNicConfig,
+} from './v2v.configs';
+import {
+  withResource,
+  createResources,
+  deleteResources,
+} from '@console/shared/src/test-utils/utils';
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { multusNAD } from '../../tests/mocks/mocks';
+import { VM_STATUS } from '../../tests/utils/constants/vm';
+import { V2V_VM_IMPORT_TIMEOUT } from '../../tests/utils/constants/common';
+
+describe('Kubevirt create VM using wizard', () => {
```

Comment:
it is not possible to run these test for me as I am getting this error:

will debug more

```

>> An error was thrown in an afterAll
   UnexpectedAlertOpenError: unexpected alert open: {Alert text : Are you sure you want to navigate away from this form? Any data you've added will be lost.}
     (Session info: chrome=84.0.4147.125)
     (Driver info: chromedriver=84.0.4147.30 (48b3e868b4cc0aa7e8149519690b6f6949e110a8-refs/branch-heads/4147@{#310}),platform=Linux 5.8.14-arch1-1 x86_64)
       at Object.checkLegacyResponse /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/error.js:553:13
       at parseHttpResponse /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/http.js:509:13
       at /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/http.js:441:30
       at processTicksAndRejections internal/process/task_queues.js:93:5
   From: Task: WebDriver.takeScreenshot()
       at Driver.schedule /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/webdriver.js:807:17
       at Driver.takeScreenshot /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/webdriver.js:1085:17
       at run /home/ansy/projects/openshift/console/frontend/node_modules/protractor/built/browser.js:59:33
       at ProtractorBrowser.to.<computed> [as takeScreenshot] /home/ansy/projects/openshift/console/frontend/node_modules/protractor/built/browser.js:67:16
       at /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/index.js:600:23
       at /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/node_modules/lodash/lodash.js:4911:15
       at baseForOwn /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/node_modules/lodash/lodash.js:2996:24
       at /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/node_modules/lodash/lodash.js:4880:18
       at Function.forEach /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/node_modules/lodash/lodash.js:9344:14
       at Jasmine2ScreenShotReporter.specDone /home/ansy/projects/openshift/console/frontend/node_modules/protractor-jasmine2-screenshot-reporter/index.js:596:7

```

Reply:
Found the reason, some my code was removed when rebasing from master, now everything is working fine.